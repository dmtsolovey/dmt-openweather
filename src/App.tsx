import { BrowserRouter, Route, Routes } from "react-router-dom";
import Home from "./components/Home/Home";
import WeatherSummary from "./components/WeatherSummary/WeatherSummary";
import "./App.module.scss";

function App() {
  return (
    <BrowserRouter>
      <Routes>
        <Route path="/" element={<Home />} />
        <Route path="location/:id" element={<WeatherSummary />} />
      </Routes>
    </BrowserRouter>
  );
}

export default App;
