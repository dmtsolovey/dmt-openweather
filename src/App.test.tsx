import App from "./App";
import shallowRenderer from "react-test-renderer/shallow";

it("renders without crashing", () => {
  const view = shallowRenderer.createRenderer();
  view.render(<App />);
  const utils = view.getRenderOutput();
  expect(utils).toMatchSnapshot();
});
