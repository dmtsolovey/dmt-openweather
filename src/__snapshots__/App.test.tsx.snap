// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<BrowserRouter>
  <Routes>
    <Route
      element={<Home />}
      path="/"
    />
    <Route
      element={<WeatherSummary />}
      path="location/:id"
    />
  </Routes>
</BrowserRouter>
`;
